gateway/README.md

```diff
@@ -2,116 +2,167 @@
 
-### Troubleshooting
+#### Cleanup
 
-- Exposing an API with authentication takes some time to work. Looking through the API resource can help. A good API should have:
+Remove the Api by using the `kubectl delete` command on the latest Api resource you have applied. For example, if you have created an Api from the `api-with-auth.yaml` file, run the following command:
 
-```yaml
-authenticationstatus
-  code: 2
+```bash
+kubectl delete -f ./lambda/api-with-auth.yaml -n {namespace}
 ```
 
+Remove a service or lambda by using the `kubectl delete` command` on the file from which the resource was created. See the example for lambda:
+
 ```bash
-kubectl get api hello -o yaml
-apiVersion: gateway.kyma.cx/v1alpha2
-kind: api
-metadata:
-  annotations:
-    kubectl.kubernetes.io/last-applied-configuration: |
-      {"apiVersion":"gateway.kyma.cx/v1alpha2","kind":"api","metadata":{"annotations":{},"labels":{"example":"serverless-lambda","function":"hello"},"name":"hello","namespace":"default"},"spec":{"authentication":[{"jwt":{"issuer":"https://dex.kyma.local","jwksUri":"http://dex-service.kyma-system.svc.cluster.local:5556/keys"},"type":"JWT"}],"hostname":"hello.kyma.local","service":{"name":"hello","port":8080}}}
-  clusterName: ""
-  creationTimestamp: 2018-05-29T08:55:04Z
-  generation: 0
-  labels:
-    example: serverless-lambda
-    function: hello
-  name: hello
-  namespace: default
-  resourceVersion: "9563"
-  selfLink: /apis/gateway.kyma.cx/v1alpha2/namespaces/default/apis/hello
-  uid: fa7805fe-631d-11e8-a42f-3e54503ab95f
-spec:
-  authentication:
-  - jwt:
-      issuer: https://dex.kyma.local
-      jwksUri: http://dex-service.kyma.local
-    type: JWT
-  hostname: hello.kyma.local
-  service:
-    name: hello
-    port: 8080
-status:
-  authenticationStatus:
-    code: 2
-    resource:
-      name: hello
-      uid: fa807328-631d-11e8-a42f-3e54503ab95f
-      version: "9562"
-  ingressStatus:
-    code: 2
-    resource:
-      name: hello-ing
-      uid: fa7d02e1-631d-11e8-a42f-3e54503ab95f
-      version: "9561"
+kubectl delete -f ./lambda/deployment.yaml -n {namespace}
+```
+
+## Troubleshooting
+
+The problem occurs when there is an Api resource with authentication enabled but after making a request without a JWT token, the received responses code is `200`.
+
+Solution 1: Wait. If the cluster is under high workload it may take a while for authentication policies to apply. If you still have that problem after a few seconds look at the Solution 2.
+
+
+Solution 2: If you did not use the default settings, there might be something wrong with the JWKS URI you provided. If you use a local Deployment of Kyma on Minikube and the internal OIDC Identity Provider such as Dex, make sure that the JWKS URI is provided as FQDN, and that it points directly to the keys endpoint e.g. http://dex-service.kyma-system.svc.cluster.local:5556/keys. Envoy sidecars must be able to resolve a domain name to the proper inside-cluster or outside-cluster IP address.
```

Comment:
``directly to the keys endpoint, for example: http://dex-service.kyma-system.svc.cluster.local:5556/keys.``